recipes/catboost/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# install using pip from the whl files on PyPI
+
+if [ `uname` == Darwin ]; then
+    if [ "$PY_VER" == "2.7" ]; then
+        WHL_FILE=https://pypi.org/packages/cp27/c/catboost/catboost-${PKG_VERSION}-cp27-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
+    elif [ "$PY_VER" == "3.4" ]; then
+        WHL_FILE=https://pypi.org/packages/cp34/c/catboost/catboost-${PKG_VERSION}-cp34-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
+    elif [ "$PY_VER" == "3.5" ]; then
+        #WHL_FILE=https://pypi.org/packages/cp35/c/catboost/catboost-${PKG_VERSION}-cp35-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
```

Comment:
Why is this one not from pypi?

Reply:
It's fixed version for python 3.5. conda-forge CI build fails with CatBoost 0.6.1/2 now. 
Will be changed to proper pypi with next release (planned on the next week).